util/dvsim.py

```diff
@@ -113,7 +113,7 @@ def main():
     parser.add_argument("-t",
                         "--tool",
                         default="",
-                        metavar="vcs|xcelium|ascentlint|...",
+                        metavar="vcs|xcelium|ascentlint|dc|...",
```

Comment:
we may want to allow picking by tool OR by flow type. this allows to use the tool specified in the target hjson.

Reply:
I think I'd prefer by flow type. I am fine with this for now, we can clean up later.

Reply:
this is also what I have been thinking. especially since you know allow picking the tool via the config files.

Reply:
carried this over to https://github.com/lowRISC/opentitan/issues/1727